src/main/java/com/passbook/sparkeighteen/service/UserService.java

```diff
@@ -37,4 +39,24 @@ public SignUpResponse signUp(@NonNull final SignUpRequest signUpRequest) throws
                 .message("Sign-up is successful.")
                 .build();
     }
+
+    public LoginResponse login (final LoginRequest loginRequest) throws Exception {
+        final Optional<UserEntity> byEmail =
+                userRepository.findByEmail(loginRequest.getEmail());
+        if (byEmail.isPresent()) {
+            return LoginResponse.builder()
+                    .message(" user login successful")
+                    .build();
+        }
```

Comment:
For now this is ok, put a TODO here for refactoring once you have exception handling layer. You should have a orElseThrow() to throw exception with a message

Reply:
ok sir, ill try